components/messaging/conversation/_conversation.scss

```diff
@@ -0,0 +1,8 @@
+/* _conversation */
+.conversation {
```

Comment:
If we don't require any unique styling this shouldn't be a component but a pattern of other components.

Reply:
Good catch - it was pre-emptively created as I worked through the build from the prototypes.